src/util/sentry.ts

```diff
@@ -15,10 +15,13 @@ const isInformationalError = (exception: unknown): boolean => {
   return logLevel === "info";
 };
 
-const beforeSend = (
-  event: Sentry.ErrorEvent,
-  hint: Sentry.EventHint,
-): PromiseLike<Sentry.ErrorEvent | null> | Sentry.ErrorEvent | null => {
+const sentryIgnoreErrors = [
+  'Object.prototype.hasOwnProperty.call(o,"telephone")',
+  'Object.prototype.hasOwnProperty.call(e,"telephone")',
+  "'get' on proxy: property 'javaEnabled' is a read-only and non-configurable data property",
+];
+
+export const beforeSend = (event: Sentry.ErrorEvent, hint: Sentry.EventHint) => {
```

Comment:
Måtte gjøre dette for å få testen til å kjøre. Fikk ingen typefeil sjølv om eg fjerna dette heller.